pkg/arvo/sys/hoon.hoon

```diff
@@ -1349,15 +1349,13 @@
         b
       ?:  (mor n.a n.b)
         ?:  =(n.b n.a)
-          [n.b $(a l.a, b l.b) $(a r.a, b r.b)]
+          b(l $(a l.a, b l.b), r $(a r.a, b r.b))
         ?:  (gor n.b n.a)
-          $(a [n.a $(a l.a, b [n.b l.b ~]) r.a], b r.b)
-        $(a [n.a l.a $(a r.a, b [n.b ~ r.b])], b l.b)
-      ?:  =(n.a n.b)
-        [n.b $(b l.b, a l.a) $(b r.b, a r.a)]
+          $(a a(l $(a l.a, b b(r ~))), b r.b)
```

Comment:
These four changed lines can be further reduced to `$(l.a $(a l.a, r.b ~), b r.b)`, etc.